tox.ini

```diff
@@ -64,9 +64,7 @@ commands =
 
          # measuring coverage is somewhat slower than not measuring coverage
          # so only do it on request.
-         coverage: coverage run -m twisted.trial {env:TAHOE_LAFS_TRIAL_ARGS:--rterrors --reporter=timing} {posargs:{env:TEST_SUITE}}
-         coverage: coverage combine
-         coverage: coverage xml
+         coverage: make -e TEST_SUITE={posargs:{env:TEST_SUITE}} test-venv-coverage
```

Comment:
If I undrestand correctly, this change plus corresponding changes in Makefile means the exit code will no longer be 0 if tests fail, which can mean coverage-using CI will be green even if tests fail.

In addition, on Windows I suspect Makefile won't work? So I would leave this file as it was before and just have the Makefile call tox, rather than other way around.

Reply:
> the exit code will no longer be 0 if tests fail

Actually, if the tests fail, it stores the exit code, then runs the coverage reports, then exits with that exit code.  So test failures should not result in green CI builds.

> on Windows I suspect Makefile won't work

Yup, this was the cause of the Windows CI failure, so I reverted that bit.  Unfortunately, that means we won't get coverage reports in CI if the tests fail.  I could put the "report coverage even if tests fail" logic into a cross-platform Python script to be called by tox, but I'm not sure that's worth it.  I can still use the `./Makefile` target to get coverage reports along with failures locally.